Containerfile

```
@@ -8,7 +8,7 @@ LABEL org.opencontainers.image.vendor="The Retrospring team"
 LABEL org.opencontainers.image.url="https://github.com/Retrospring/retrospring"
 
 ARG RETROSPRING_VERSION=2023.0131.1
-ARG RUBY_VERSION=3.1.2
+ARG RUBY_VERSION=3.2.2
 ARG RUBY_INSTALL_VERSION=0.9.0
```

could also upgrade this one while we're at it
```suggestion
ARG RUBY_INSTALL_VERSION=0.9.2
```